draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2941,21 +2942,30 @@ https://video.example.com/k/movie1/4/013.ts
 		</list>
 
 	</t>
+
+        <t>
+        uCDN may combine multiple options in the same trigger command. dCDN should consider the following when processing such commands:
+        <list style="symbols">
+        <t>dCDN MUST reject triggers that are dependent on other pending triggers with lower priority to prevent deadlocks.</t>
+        <t>When introducing an urgent trigger uCDN SHOULD indicate a priority that is equal to or higher than the highest priority among the pending triggers that belong to the same uCDN. dCDN MUST reject urgent triggers that have lower priority than other pending triggers of the same uCDN.</t>
+        <t>dCDN MUST reject an urgent trigger that has pending triggers as its prerequisite. dCDN MAY reject an urgent trigger that has triggers in “active” or “cancelling” state as its prerequisite, such as when such triggers are deemed to take a long time to complete or cancel.</t>
+        </list>
+        </t>
+
 	<t>
-	uCDN may combine multiple options in the same trigger command. dCDN should consider the following when processing such commands:
+	When combining ExtensionPolicy with <xref target="time-policy"/>, TimePolicy governs the time window when the trigger can execute, while ExecutionPolicy controls the order of trigger execution within their respective windows.  The following error conditions should be considered by dCDN:
 	<list style="symbols">
-	<t>(i) dCDN MUST reject triggers that are dependent on other pending triggers with lower priority to prevent deadlocks.</t>
-	<t>(ii) When introducing an urgent trigger uCDN SHOULD indicate a priority that is equal to or higher than the highest priority among the pending triggers that belong to the same uCDN. dCDN MUST reject urgent triggers that have lower priority than other pending triggers of the same uCDN.</t>
-	<t>(iii) dCDN MUST reject an urgent trigger that has pending triggers as its prerequisite. dCDN MAY reject an urgent trigger that has triggers in “active” or “cancelling” state as its prerequisite, such as when such triggers are deemed to take a long time to complete or cancel.</t>
+		<t>When a trigger is marked urgent, but TimePolicy doesn't allow immediate processing start, it should be rejected by dCDN</t>
```

Comment:
I suggest removing "start" as it seems redundant.  Also, ending the sentence with a period would make it consistent with the bullet list above.